nf_wochenende.nf

```diff
@@ -224,14 +224,6 @@ workflow {
     //haybaler(reporting.out.us_csvs.collect().flatten())
```

Comment:
I always forget to remove this unnessecary line, could you do that?

Reply:
done

Reply:
Thanks!